src/deploy/extensions/planner.ts

```diff
@@ -215,17 +215,28 @@ export async function want(args: {
  */
 export async function resolveVersion(ref: refs.Ref): Promise<string> {
   const extensionRef = refs.toExtensionRef(ref);
+  const extension = await extensionsApi.getExtension(extensionRef);
+  if (!ref.version || ref.version === "latest-approved") {
+    if (!extension.latestApprovedVersion) {
+      throw new FirebaseError(
+        `${extensionRef} has not been published to https://extensions.dev. To install it, you must specify the version you want to install.`
```

Comment:
Hmm, maybe would be good to mention it by name to start building the brand. Maybe like `has not been published to Extensions Hub (https://extensions.dev)`?